cases/util/ensureCORS.js

```diff
@@ -0,0 +1,24 @@
+import { fetch } from "./fetchShim";
+
+export const ensureCORS = async (URL, httpVerb) => {
```

Comment:
for clarity, we can make httpVerb default to GET (it seems like thats the only thing used anyway)

Reply:
`async (URL, httpVerb = "GET")`